cliquet/resource.py

```diff
@@ -239,7 +238,8 @@ class BaseResource(object):
     mapping = ResourceSchema()
     """Schema to validate records."""
 
-    def __init__(self, request):
+    def __init__(self, request, context=None):
```

Comment:
Hmm, and I misread this it seems! You probably are right and we don't need to have this `context=None` here, I can just remove the default argument. /me tries.